src/commcare_cloud/commands/ansible/service.py

```diff
@@ -274,10 +276,57 @@ class Nginx(AnsibleService):
     inventory_groups = ['proxy']
 
 
-class Elasticsearch(AnsibleService):
+class ElasticsearchClassic(AnsibleService):
+    name = 'elasticsearch-classic'
+    service_name = 'elasticsearch'
+    inventory_groups = ['elasticsearch']
+
+
+class Elasticsearch(ServiceBase):
     name = 'elasticsearch'
+    service_name = 'elasticsearch'
     inventory_groups = ['elasticsearch']
 
+    def execute_action(self, action, host_pattern=None, process_pattern=None):
+        if action == 'status':
+            return ElasticsearchClassic(self.environment, self.ansible_context).execute_action(action, host_pattern, process_pattern)
+        else:
+            if not ask(
+                    "This function does more than stop and start the elasticsearch service. "
+                    "For that, use elasticsearch-classic."
+                    "\nStop will: stop pillows, stop es, and kill -9 if any processes still exist "
+                    "after a period of time. "
+                    "\nStart will start pillows and start elasticsearch "
+                    "\nRestart is a stop followed by a start.\n Continue?", strict=False):
+                return 0  # exit code
+            if action == 'stop' or action == 'restart':
+                exit_code = self._act_on_pillows(action=action)
+                if not exit_code == 0:
+                    print("ERROR while stopping pillows. Exiting.")
+                    sys.exit(1)
+                self._run_rolling_restart_yml(tags='action_stop')
+            if action == 'start' or action == 'restart':
+                exit_code = self._act_on_pillows(action=action)
+                if not exit_code == 0:
+                    print("ERROR while starting pillows. Exiting.")
+                    sys.exit(1)
+                self._run_rolling_restart_yml(tags='action_start')
```

Comment:
it looks like elasticsearch is being started after pillows are started if i'm reading this correctly. we probably want to start elasticsearch first, and then start pillows

Reply:
How come we want to do it in that order?

Reply:
most pillows populate data into elasticsearch, so for the time between the pillows starting and es starting, the pillows will just be creating errors. they will retry eventually but there's no point to starting a service if the service it depends on is down

Reply:
That makes sense, done here: https://github.com/dimagi/commcare-cloud/pull/1990/commits/9a8c2aeaf2db8f0b89756856d6bbe0301120dfba
(also refactored the pillow start/stop code in there)